src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -216,19 +251,39 @@ function MoneyRequestParticipantsSelector({
         chatOptions.userToInvite,
         personalDetails,
         translate,
-        cleanSearchTerm,
+        showImportContacts,
+        inputHelperText,
     ]);
 
+    const handleContactImport = useCallback(() => {
+        contactImport().then(({contactList, permissionStatus}: ContactImportResult) => {
+            setContactPermissionState(permissionStatus);
+            const usersFromContact = getContacts(contactList);
+            setContacts(usersFromContact);
+        });
+    }, []);
+
+    useEffect(() => {
+        getContactPermission().then((status) => {
+            setContactPermissionState(status);
+            if (status !== RESULTS.BLOCKED && status !== RESULTS.UNAVAILABLE) {
+                setSoftPermissionModalVisible(true);
+            }
+        });
+    }, []);
+
     /**
      * Adds a single participant to the expense
      *
      * @param {Object} option
      */
     const addSingleParticipant = useCallback(
-        (option: Participant) => {
+        (option: Participant & OptionsListUtils.Option) => {
+            const participantDetails = lodashPick(option?.participantsList?.at(0), 'firstName', 'lastName', 'displayName', 'phoneNumber', 'avatar');
```

Comment:
This is option object. Instead of creating `participantDetails` variable, we can simplify it by adding  text and phoneNumber fields to the below lodashPick
```
{
    "accountID": 1025377084,
    "alternateText": "0387025654",
    "brickRoadIndicator": null,
    "hasDraftComment": false,
    "icons": [
        {
            "id": 1025377084,
            "name": "0387025654",
            "source": 271,
            "type": "avatar"
        }
    ],
    "iouReportAmount": 0,
    "isBold": true,
    "isChatRoom": false,
    "isDefaultRoom": false,
    "isExpenseReport": false,
    "isIOUReportOwner": false,
    "isOptimisticAccount": true,
    "isOwnPolicyExpenseChat": false,
    "isPinned": false,
    "isPolicyExpenseChat": false,
    "isWaitingOnBankAccount": false,
    "item": {
        "accountID": 1025377084,
        "avatar": 271,
        "displayName": "Dyaln Do",
        "firstName": "Dyaln",
        "lastName": "Do",
        "login": "0387025654",
        "phoneNumber": "0387025654",
        "pronouns": "",
        "validated": true
    },
    "keyForList": "1025377084",
    "lastMessageText": "",
    "login": "0387025654",
    "participantsList": [
        {
            "accountID": 1025377084,
            "avatar": 271,
            "displayName": "Dyaln Do",
            "firstName": "Dyaln",
            "lastName": "Do",
            "login": "0387025654",
            "phoneNumber": "0387025654",
            "pronouns": "",
            "validated": true
        }
    ],
    "phoneNumber": "0387025654",
    "reportID": "",
    "shouldAnimateInHighlight": false,
    "shouldShowSubscript": false,
    "text": "Dyaln Do",
    "tooltipText": null
}


```

Reply:
Sure, I'll merge it